demo/api/src/form-builder/blocks/text-input.block.ts

```diff
@@ -1,19 +1,38 @@
 import { BlockField, createBlock, inputToData } from "@comet/blocks-api";
-import { IsOptional, IsString } from "class-validator";
+import { IsEnum, IsOptional, IsString } from "class-validator";
 
 import { BaseFieldBlockData, BaseFieldBlockInput } from "./base-field.block";
 
+const inputTypes = ["text", "email", "phone", "number"];
+
+type InputType = (typeof inputTypes)[number];
```

Comment:
Why not use a real enum? 😅

Reply:
I don't want to define and maintain both a key and a value 😄